lib/fungus_toast/games/games.ex

```diff
@@ -244,13 +246,15 @@ defmodule FungusToast.Games do
   @doc """
   Executes a full round of growth cycles and creates a new round for this game
   """
-  def trigger_next_round(game_id) do
+  def trigger_next_round(game_id, grid_size) do
```

Comment:
Can we not calculate the grid size with game info? There doesn't seem to be a reason to actually pass the value in when you can figure it out based on information in the Game object. In this case, I'd probably use the ID to look up the `Game` and pass that to another `trigger_next_round` that takes a `Game` struct and then do this work

Reply:
Yeah, since there is a game already available when I call this (AND players is already preloaded) I should definitely do this. On it.